src/test/resources/org/datadog/jmxfetch/util/server/run-SimpleApp.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
+set -f
+
+[ -n "$JAVA_OPTS" ] || JAVA_OPTS="-Xmx128M -Xms128M"
+[ -n "$RMI_PORT" ] || RMI_PORT="9010"
+
+echo "Using `java --version`"
+echo "With JAVA_OPTS '${JAVA_OPTS}'"
+CONTAINER_IP=`awk 'END{print $1}' /etc/hosts`
```

Comment:
TIL awk keeps the last line's vars around for access in `END` https://www.gnu.org/software/gawk/manual/html_node/I_002fO-And-BEGIN_002fEND.html